src/framework/application.js

```diff
@@ -150,6 +150,15 @@ Object.assign(pc, function () {
         // Store application instance
         Application._applications[canvas.id] = this;
         Application._currentApplication = this;
+
+        /**
+         * @field
+         * @static
+         * @readonly
+         * @type {pc.Application|Undefined}
+         * @name pc.app
```

Comment:
This is actually private. The officially supported way to get the current application is pc.Application.getApplication.

Reply:
Alright, made it private.